src/main/java/ch/njol/skript/sections/LoopSection.java

```diff
@@ -33,52 +40,52 @@
 import java.util.Map;
 import java.util.WeakHashMap;
 
-/**
- * A trigger section which represents a loop.
- * 
- * @author Peter Güttinger
- */
-public class Loop extends TriggerSection {
+public class LoopSection extends CodeSection {
 
-	private final Expression<?> expr;
+	static {
+		Skript.registerSection(LoopSection.class, "loop %objects%");
```

Comment:
Does https://github.com/SkriptLang/Skript/blob/master/src/main/java/ch/njol/skript/expressions/ExprTimes.java need to be edited too?

Reply:
I don't think so